style.css

```diff
@@ -68,6 +69,195 @@ body {
   .social-icon {
     padding-right: 6%;
   }
+
+  .about {
+    width: 375px;
+    background-color: white;
+    margin-top: 15%;
+  }
+
+  .about-container {
+    margin-left: 5%;
+    margin-right: 5%;
+    padding-top: 15%;
+  }
+
+  .resume {
+    width: 150px;
+    height: 30px;
+    font-family: "Poppins", sans-serif;
+    font-weight: bolder;
+    font-size: 17px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    border: 2px solid #6070ff;
+    border-radius: 8px;
+    padding: 12px;
+    gap: 10px;
+  }
+
+  .language {
+    display: flex;
+  }
+
+  .l-languages {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: bolder;
+    line-height: 24px;
+    gap: 145px;
+    margin-top: 7%;
+  }
+
+  .f-languages {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: bolder;
+    line-height: 24px;
+    gap: 124.5px;
+    border-color: #e5e5e5;
+    border-style: solid;
+    border-top: 0;
+    border-right: 0;
+    border-left: 0;
+    margin-top: 7%;
+  }
+
+  .s-languages {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+    font-weight: bolder;
+    line-height: 24px;
+    gap: 200px;
+    border-color: #e5e5e5;
+    border-style: solid;
+    border-top: 0;
+    border-right: 0;
+    border-left: 0;
+    margin-top: 7%;
+  }
+
+  .techstack {
+    display: flex;
+    list-style: none;
+    flex-direction: column;
+    padding-left: 0%;
+  }
+
+  .files {
+    display: flex;
+    margin-top: 5%;
+    background-color: #e5e5e5;
```

Comment:
- [ ] Kindly use the correct background color to be similar to the Figma design 👇🏽 

`background: #F7F7F9;`